.eslintrc.json

```diff
@@ -14,11 +14,14 @@
         "ShadyCSS": false
     },
     "rules": {
-        "brace-style": [2, "1tbs", { 
-            "allowSingleLine": true 
+        "array-bracket-newline": [2, { "minItems": 2 }],
```

Comment:
Require newline after opening/before closing array bracket if there are 2 or more items (used in conjunction with `array-element-newline` below).

*Good*
```javascript
var a = [ 'one item' ];
var b = [
    'first item',
    'second item',
];
```

*Bad*
```javascript
var b = [ 'first item', 'second item' ];
```

Reply:
looks correct but shouldn't it have added new lines on
``` javascript
static get observedAttributes () {
       return ['open','position','trigger'];
}
```
in `hxTooltip.js`?